sdk/Sdk/FunctionMetadataGenerator.cs

```diff
@@ -428,15 +426,20 @@ private static ExpandoObject BuildBindingMetadataFromAttribute(CustomAttribute a
             bindingDict["Type"] = bindingType;
             bindingDict["Direction"] = GetBindingDirection(attribute);
 
+            // Is sdk parameter type
```

Comment:
nit - adjust comment to "For extensions that support worker binding, set the DataType so all parameters are bound by the worker."

Reply:
Done